tests/test_array.py

```diff
@@ -218,3 +219,23 @@ def test_warning_missing_placeholder_in_output(hq_env: HqEnv, channel: str):
         in output
     )
     assert f"Consider adding `%{{TASK_ID}}` to the `--{channel}` value."
+
+
+def test_array_times(hq_env: HqEnv):
+    hq_env.start_server()
+    hq_env.start_worker()
+
+    hq_env.command(["submit", "--array=1-3", "sleep", "1"])
+    wait_for_job_state(hq_env, 1, "FINISHED")
+
+    time.sleep(
+        1.2
+    )  # This sleep is not redundant, we check that after finished time is not moving
+
+    table = hq_env.command(["job", "1", "--tasks"], as_table=True)
+    offset = JOB_TABLE_ROWS
+    for i in range(1, 4):
+        start = datetime.datetime.strptime(table[offset + i][3][0:-7], "%Y-%m-%d %H:%M:%S.%f")
+        end = datetime.datetime.strptime(table[offset + i][4][0:-7], "%Y-%m-%d %H:%M:%S.%f")
+        seconds = (end - start).total_seconds()
+        assert 1.1 > seconds > 0.9
```

Comment:
```suggestion
        assert 0.9 <= seconds <= 1.1
```
Seems a bit more readable :)